### PR TITLE
[INLONG-3884][Sort] Add CSV format for ExtractNode and LoadNode

### LINKED ISSUE #3884: [Improve][Sort] Add CSV format for ExtractNode and LoadNode and optimize fill options of table
### Description

Add csv format for ExtractNode and LoadNode and optimize fill options of table

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### [INLONG-3884][Sort]Add csv format for ExtractNode and LoadNode and optimize fill options of table

Fixes #3884 

### Motivation

It will support csv format

### Modifications

Add csv format class.
Optimize fill options of table

*Describe the modifications you've done.*

### CODE REVIEW

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/CsvFormat.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        return "csv";
+    }
+
+    /**
+     * Generate options for connector
+     * @param includePrefix true will need append key and value when format is json avro csv
+     * @return options
+     */
+    @Override
+    public Map<String, String> generateOptions(boolean includePrefix) {
```
Comment (EMsnap) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/CsvFormat.java:94]: method is too long

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/JsonFormat.java [hunk 2]
```diff
@@ -88,35 +88,54 @@ public String getFormat() {
 
     /**
      * Generate options for connector
-     *
+     * @param includePrefix true will need append key and value when format is json avro csv
      * @return options
      */
     @Override
-    public Map<String, String> generateOptions() {
+    public Map<String, String> generateOptions(boolean includePrefix) {
```
Comment (EMsnap) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/JsonFormat.java:95]: method is too long

Comment (EMsnap) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/JsonFormat.java:95]: method is too long

Comment (gong) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/JsonFormat.java]: ok

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/CsvFormat.java (hunk 1 above)
Comment (gong) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/CsvFormat.java]: ok

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Pull request merged**